EssentialFeed/EssentialFeed/Image Comments/API/ImageCommentsEndpoint.swift

```diff
@@ -0,0 +1,16 @@
+//
+// Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import Foundation
+
+public enum ImageCommentsEndpoint {
+	case get(imageID: String)
```

Comment:
The API specs define the ImageID as a UUID, so ideally this API should enforce that instead of defining the imageID as a String.

This way we can prevent wrong input like the string: `ImageCommentsEndpoint.get("this is not a valid id")`

I recommend you change String to UUID:

```suggestion
	case get(imageID: UUID)
```